.travis.yml

```diff
@@ -45,74 +45,6 @@ matrix:
         - ICC1=gcc-6
         - ICXX1=g++-6
 
-    - os: linux
-      addons:
-        apt:
-          sources:
-            - ubuntu-toolchain-r-test
-          packages:
-            - g++-6
-      env:
-        - FAVORITE_CONFIG=no
-        - LLVM_VERSION="3.8.1"
-        - LLVM_CONFIG="llvm-config-3.8"
-        - config=debug
-        - CC1=gcc
-        - CXX1=g++
-        - ICC1=gcc-6
-        - ICXX1=g++-6
-
-    - os: linux
-      addons:
-        apt:
-          sources:
-            - ubuntu-toolchain-r-test
-          packages:
-            - g++-6
-      env:
-        - FAVORITE_CONFIG=no
-        - LLVM_VERSION="3.8.1"
-        - LLVM_CONFIG="llvm-config-3.8"
-        - config=release
-        - CC1=gcc
-        - CXX1=g++
-        - ICC1=gcc-6
-        - ICXX1=g++-6
-
-    - os: linux
-      addons:
-        apt:
-          sources:
-            - ubuntu-toolchain-r-test
-          packages:
-            - g++-6
-      env:
-        - FAVORITE_CONFIG=no
-        - LLVM_VERSION="3.9.1"
-        - LLVM_CONFIG="llvm-config-3.9"
-        - config=debug
-        - CC1=gcc
-        - CXX1=g++
-        - ICC1=gcc-6
-        - ICXX1=g++-6
-
-    - os: linux
```

Comment:
This is actually a mistake on my part. This would break release builds. Would need to do something to address. So we run a linux build that exits out if not on RELEASE.